core/partitioning/partitioning.cpp

```diff
@@ -200,6 +200,10 @@ void resolveNonTensorInputs(PartitionedGraph& segmented_blocks) { // , std::shar
   }
   segmented_blocks.clear();
   segmented_blocks.insert(segmented_blocks.begin(), segmented_blocks_list.begin(), segmented_blocks_list.end());
+  PartitionCtx partition_context;
```

Comment:
How urgent is this change? I think we might as well start moving the PartitionGraph object, the IValue maps etc. into the context now and pass this one object around in this patch vs just leaving it for later

Reply:
I don't think this one is so urgent, we could leave it for later.

Reply:
What I mean is we should put in the work now to move everything into the context as long as it isnt blocking you instead of making half a patch